src/client/game_board/remote/board_client.ts

```diff
@@ -67,6 +67,10 @@ export class BoardClient {
         }
         reject(new Error('Received invalid board model!'));
       });
+      this.socket.on(Events.BOARD_GET_ERROR, (ex) => {
+        console.log(`Error thrown: ${ex}`);
+        reject(ex);
+      });
```

Comment:
Possibly, add a test case for this?

Reply:
Nice... caught an issue where I was just rejecting with a string instead of an Error